source/includes/_egress.md

```diff
@@ -2,119 +2,76 @@
 
-### Response JSON Fields
-
-Field | Type | Description
---------- | ------- | -----------
-invoice_id | String | Internal **ON/RAMP**'s Invoide Identifier
-invoice_url | String | Internal **ON/RAMP**'s URL Invoice Reference for that **USR**
+Field             |   Type      | Description
+----------------- | ----------- | ---------
 
-## Callback Egress Invoice
-
-> Example Callback 
-
-```shell
-curl "https://callback.url/ack_webhook?param1=value1&param2=value2&invoice_id=62b570e8-9723-4287-a5a8-e825c2ffced2&issued_at=1565184896401"
-```
 
-> Response should be 200, otherwise system will retry with exponential decay 5 times
+### Callback Egress Invoice
 
-
-Callback Egress Invoice, is the callback **ON/RAMP** system is going to call back to the **URL** set in [Create Egress Invoice](#create-egress-invoice) endpoint on the field `payment_ack_url`, once the Egress has been successfully completed. This callback will be done with exactly the same **URL** setup in `payment_ack_url`. This means that if `payment_ack_url** contains Query Parameters in the URL it will be maintain.
-
-Additionally with your `payment_ack_url`, **ON/RAMP** will add some **Query Parameters** that will overwrite original `payment_ack_url` query parameters if they are the same. This **ON/RAMP** query parameters are like *reserved word* parameters keys with information about the invoice which is being callback from.
-
-**URL** in **cURL** tab is an example and is the same put it as an example in field ``payment_ack_url` on [Create Egress Invoice](#create-egress-invoice) section.
-
-
-### Reserved Query Parameters
-
-Parameter | Type | Description
---------- | ------- | -----------
-invoice_id | String | **USR**'s invoice id returned in [Create Egress Invoice](#create-egress-invoice) endpoint
-issued_at | Number | Invoice completion Timestamp
-
-## Query Egress Invoice
-
-> Example Call
-
-```shell
-curl "https://api.onramp.com/v1/egress/invoice/62b570e8-9723-4287-a5a8-e825c2ffced2" \
--H "Authorization: Bearer AUTH_TOKEN" 
-```
-
-> Response JSON
-
-```json
-{
-  "invoice_id": "62b570e8-9723-4287-a5a8-e825c2ffced2",
-  "invoice_url": "https://api.onramp.com/egress/62b570e8-9723-4287-a5a8-e825c2ffced2",
-  "status": "processing"
-}
-```
-
-### HTTP Request
-
-<aside class="success"><b><code>GET /v1/egress/invoice/:id</code></b></aside>
-
-### Query Parameter Path
-
-Field | Type | Description
---------- | ------- | -----------
-id | String | **USR**'s invoice id returned in [Create Egress Invoice](#create-egress-invoice) call
+Once **ON/RAMP** is ready to commit the payment, it will call back merchant to confirm. **It is only at this point
+when the user transaction should be consider done and be debited at merchant's side**. If the merchant was able to
+debit the user, then it should answer the callback with an http 200 error code; if the merchant was unable to
+debit the user, it should answer with an http 204 error code (this will cancel the user payment, expiring his
```

Comment:
No, the idea was to use a non-error code (it is not an error, it is just answering back what should be done) that were uncommon, to avoid interpreting unexpected problems (i.e wrong configuration or some server being down) as "no-ok" answers.